projects/tags-input-demo/src/app/app.component.ts

```diff
@@ -49,6 +50,24 @@ export class AppComponent {
     ddHasBackdrop: false
   };
 
+
+  ImageTagsInputConfig: AngularTagsInputConfig = {
```

Comment:
The variable name is in the capital. Should be camelCase

Reply:
fixed